src/config/config.cpp

```diff
@@ -163,7 +163,7 @@ FullNodeConfig::FullNodeConfig(Json::Value const &string_or_object, Json::Value
     if (auto path = getConfigData(root["logging"], {"log_path"}, true); !path.isNull()) {
       log_path = path.asString();
     } else {
-      log_path = db_path / "logs";
+      log_path = db_path / "../logs";
```

Comment:
Maybe we should have something like `datadir_path` filed in that config?

`auto net_file_path() const { return db_path / "net"; }`
There is that method in a header, so looks like the same problem with net_path

Reply:
agree